### PR TITLE
Added Support for AMP as specified in issue #218

### PR DESCRIPTION
**Pull request by rohittm**

### CODE REVIEW

**Comment by HardeepAsrani** [comment]
@rohittm Why this? https://github.com/rohittm/wp-product-review/blob/f479bbdb6342995a7215f24edbf9f81075713813/includes/public/class-wppr-public.php#L296

Use require_once instead of include_once: https://github.com/rohittm/wp-product-review/blob/f479bbdb6342995a7215f24edbf9f81075713813/includes/public/class-wppr-public.php#L632

**Comment by rohittm** [comment]
Removed this:  https://github.com/rohittm/wp-product-review/blob/f479bbdb6342995a7215f24edbf9f81075713813/includes/public/class-wppr-public.php#L296 , Check the last commit.

 https://github.com/rohittm/wp-product-review/blob/f479bbdb6342995a7215f24edbf9f81075713813/includes/public/class-wppr-public.php#L632 , replacing it. I saw this from: https://codex.wordpress.org/Function_Reference/is_plugin_active

**Comment by contactashish13** [comment]
@rohittm please make the changes I had suggested in our last discussion about following OOP methodology correctly.

**Comment by rohittm** [comment]
@contactashish13 Can you check and review the code now?

**Comment by contactashish13** [comment]
@rohittm are both functions required to test the presence of the AMP plugin? I think it can be reduced to one, unless there is a specific reason.

**Comment by HardeepAsrani** [comment]
@contactashish13 There are two major plugins used for AMP, one is by Automattic and another one is AMP for WP, do Rohit is checking if either of them is activated. They don't have a shared function or constant name.

File: includes/public/class-wppr-public.php [hunk 1]
```diff
@@ -450,8 +459,18 @@ public function show_comment_ratings( $text ) {
...
 		$return .= '</div>';
-
 		return $return . $text . '<div class="cwpr_clearfix"></div>';
 	}
 
+	/**
+	 * AMP support for WPPR
+	 */
+	public function wppr_amp_support() {
+		$output = '';
```
Comment (selul) [inline includes/public/class-wppr-public.php]: @rohittm i'm not sure why you are not enqueue using wp_enqueue_style  ?

Comment (rohittm) [reply includes/public/class-wppr-public.php]: @selul We cannot add styles in AMP, other than the AMP one. So, I am adding the styles needed for the plugin with the AMP standards. Also, the styles generate colors on the basis of ratings, so we cannot enqueue it directly.

Comment (selul) [reply includes/public/class-wppr-public.php]: got it. i will check this out and get back to you.

**Pull request merged**